website/static/js/profile.js

```diff
@@ -539,9 +581,21 @@ var SocialViewModel = function(urls, modes) {
             {label: 'Google Scholar', text: self.scholar(), value: self.scholar.url()}
         ];
     });
-
+        
+    self.profileWebsiteEmpty = ko.computed(function() {
```

Comment:
There should never be any blank websites. A user's `social.profileWebsites` field should only contain non-empty, valid values.